src/containers/AIEvals/AIEvaluationList/AIEvaluationList.module.css

```
@@ -0,0 +1,125 @@
+  color: #3d3d3d;
+}
+
+.Dot {
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+  flex-shrink: 0;
+}
+
+.DotCompleted {
+  composes: Dot;
```

Stylelint (17.10.0)

[error] Expected "Dot" to be "dot" (value-keyword-case)

(value-keyword-case)